NotePad/src/main/java/org/openintents/notepad/NoteEditor.java

```diff
@@ -498,6 +498,17 @@ protected void onCreate(Bundle savedInstanceState) {
         // res/layout/note_editor.xml
         setContentView(R.layout.note_editor);
 
+        // now that we have the layout, finish actionbar setup
+        if (mActionBarAvailable) {
+            setSupportActionBar((Toolbar) findViewById(R.id.oi_toolbar));
+            WrapActionBar bar = new WrapActionBar(this);
+            bar.setDisplayHomeAsUpEnabled(true);
+            // force to show the actionbar on version 14+
+            if (Integer.valueOf(android.os.Build.VERSION.SDK) >= 14) {
```

Comment:
This is true for all builds as the minSdk is 14, furthermore android.os.Build.VERSION.SDK_INT should be used.

Reply:
Thank you, good point. I've cleaned this up now.